src/Nethermind/Nethermind.Facade/Eth/BlockForRpc.cs

```diff
@@ -30,7 +30,6 @@ protected BlockForRpc()
     public BlockForRpc(Block block, bool includeFullTransactionData, ISpecProvider specProvider)
     {
         _isAuRaBlock = block.Header.AuRaSignature is not null;
-        Author = block.Author ?? block.Beneficiary;
```

Comment:
Change to 
`Author = block.Author;`

Then let `[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]` remove it?

Reply:
I like that better!